README.md

```
@@ -1,2 +1,430 @@
+POST /api/users/follow/{userId}    # 팔로우/언팔로우
+POST /api/posts/{id}/like         # 좋아요/좋아요 취소
+POST /api/posts/{id}/comments     # 댓글 작성
+```
+
+모든 API 엔드포인트와 상세 스펙은 **[API 명세서](https://troublog.shop/swagger-ui/index.html#/)** 에서 확인하실 수 있습니다.
+
+---
+
+# ❗ 자주 발생하는 문제
+
+### 🔑 JWT Secret Key Missing
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)